.github/workflows/ci.yml

```diff
@@ -57,3 +57,18 @@ jobs:
 
       - name: Build 🚧
         run: pnpm run build
+
+  build-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build docker image
+        uses: docker/build-push-action@v4
+        with:
+          push: false
+          tags: rotki/frontend:latest
```

Comment:
The build is just to make sure that whatever changes in the frontend don't break the image.

We don't push to any registry because this happens only during deployment and we push to a private registry. The reason we do the build in this repo now is because previously the Dockerfile was part of the infra repo, and sometimes the build will break because of changes in this repo but they would only be detected during deployment.